src/helpers/helpers.dom.js

```diff
@@ -79,6 +84,13 @@ function getCanvasPosition(evt, canvas) {
 }
 
 export function getRelativePosition(evt, chart) {
+  if ('native' in evt) {
+    return {
```

Comment:
Would maybe be a little faster to just return the `evt` here. @etimberg any objections to that?

Reply:
no objections